_posts/2023-09-13-prospero-add-feature-pack.adoc

```diff
@@ -0,0 +1,149 @@
+---
+layout: post
+title:  "Installing additional feature packs in WIldfly"
+date:   2023-09-13
+tags:   prospero galleon wildfly
+author: spyrkob
+description: Using Prospero to installing additional feature packs in existing Wildfly servers.
+---
+
+== Additional feature packs for Wildfly
+
+Wildfly server capabilities can be extended by installing additional feature packs such as wildfly-datasources-galleon-pack or wildfly-myfaces-galleon-pack.
+The feature packs can be installed using Galleon or Prospero.
+
+In this article we will use Prospero to install a MariaDB datasource provided by wildfly-datasources-galleon-pack.
```

Comment:
related to my "breadcrumb" comment above, insists that it's installing to an existing unzipped download of WildFly. something like: "install a MariaDB datasource provided by wildfly-datasources-galleon-pack to an existing WildFly server (downloaded from the server zip archive".